packages/Python/lldbsuite/test/lang/swift/resilience/TestResilience.py

```diff
@@ -71,10 +71,10 @@ def setUp(self):
         TestBase.setUp(self)
 
     def createSymlinks(self, exe_flavor, mod_flavor):
-        execute_command("ln -sf " + self.getBuildArtifact("main." + exe_flavor) + " " + self.getBuildArtifact("main"))
+        execute_command("cp " + self.getBuildArtifact("main." + exe_flavor) + " " + self.getBuildArtifact("main"))
         execute_command("ln -sf " + self.getBuildArtifact("main." + exe_flavor + ".dSYM") + " " + self.getBuildArtifact("main.dSYM"))
 
-        execute_command("ln -sf " + self.getBuildArtifact("libmod." + exe_flavor + ".dylib") + " " + self.getBuildArtifact("libmod.dylib"))
+        execute_command("cp " + self.getBuildArtifact("libmod." + exe_flavor + ".dylib") + " " + self.getBuildArtifact("libmod.dylib"))
```

Comment:
why this?

Reply:
When the platform code tries to copy things to the remote, it deals correctly with actual files, but it's unclear what it should do with symbolic links. Should it recreate a link on the remote side or create a regular file with the contents of the pointed file?
Rather than implement a bunch of stuff on the platform side and deal with policy, doing a copy just side-stepped these issues with no downside that I can see.